### PR TITLE
Seurat's style art 

### PR DESCRIPTION
**Pull request by HarrisMBrad**
We embark on an artistic adventure. Seurat's style, mirroring, we shall, with pointillistic precision. In the intricate world of meticulously placed dots and vibrant colors, we shall immerse ourselves, ready to forge a truly unique and captivating piece. 
A harmonious collaboration between human creativity and AI's artistic prowess shall bring forth this creation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,3 +124,5 @@ as they have to review the complete pull request again.
 If you don't follow this rule, maintainers reserve the right to mark your pull request as invalid.
 
 **In this case it won't count for Hacktoberfest**.
+
+[![An image of @7597243's Holopin badges, which is a link to view their full Holopin profile](https://holopin.me/7597243)](https://holopin.io/@7597243)
```
Comment (sophiabrandt) [inline README.md:128]: Please remove this addition.

File: Art/Groot-2001/style.css [hunk 2]
```diff
@@ -1,42 +1,26 @@
...
-    transform: rotate(360deg);
+@keyframes flip {
+  0% {
+    transform: scaleY(1);
+  }
+
+  50% {
+    transform: scaleY(-1);
   }
 }
```
Comment (sophiabrandt) [inline Art/Groot-2001/style.css:26]: Please rollback changes to this file.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

**Comment by sophiabrandt** [comment]
Closing this due to inactivity.

**Comment by sophiabrandt** [comment]
@HarrisMBrad are you still interested in getting this merged? It looks like my comments above were not yet addressed.

If you solved those points, please mark the discussions as resolved.

**Comment by HarrisMBrad** [comment]
Sure, if it's not merging, it's probably my fault. Formatting for this was my issue.  Is it mergable?

**Comment by sophiabrandt** [comment]
Not currently, no. You would need to fix the issues discussed above in the comments.

**Comment by l-white** [comment]
@HarrisMBrad The last several commits have been empty, and given the complicated history of commits, I would advise starting a new branch and submitting your changes. If you have any questions, please feel free to ask for help on the ZTM Discord server. It can take time and practice to learn version control, and I encourage you to keep trying.

**Comment by HarrisMBrad** [comment]
Ok... I might do that. Thanks for any advice. It was the .yml or formatting that I can't seem to get right.

**Pull request closed without merging**